usr/src/uts/common/fs/nfs/nfs4_srv.c

```diff
@@ -250,6 +251,8 @@ static nfsstat4 check_open_access(uint32_t,
 				struct compound_state *, struct svc_req *);
 nfsstat4 rfs4_client_sysid(rfs4_client_t *, sysid_t *);
 void rfs4_ss_clid(rfs4_client_t *);
+void rfs_log_deletions(int rfs_version, char *name, struct svc_req *req,
```

Comment:
Stylistically most of the other callers here don't have args in the prototype. Is there a reason this is different?